src/test/ui/conditional-compilation/cfg_accessible-bugs.rs

```diff
@@ -0,0 +1,19 @@
+// check-fail
+// known-bug
```

Comment:
It was a really bad idea to allow skipping `//~ ERROR` annotations in `// known-bug` test.
If `//~ ERROR` annotations cannot be added together with `// known-bug`, then `// known-bug` is better removed here.